dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/webapp/WebAppConfiguration.java

```diff
@@ -2169,4 +2169,38 @@ public void setModuleResponseEncoding(String encoding) {
         }
         this.responseEncoding = encoding;
     }
+    
+    /**
+     * Since Servlet 6.0 - opt-out verifying the encoded char in URI.
+     * 
+     * true - skip checking for %23 , %2e , %2f , %5c in URI
+     */
+    public void setSkipEncodedCharVerification() {
```

Comment:
We only care about this context-parameter and metaType in Servlet 6.0 correct? Where are we verifying that it isn't used when not using the servlet-6.0 feature?

Reply:
this setSkipEncodedCharVerification() is only called from the new 
WebAppConfiguratorHelper60() which is active only for servlet6.0+.